.drone.yml

```diff
@@ -0,0 +1,44 @@
+---
+kind: pipeline
+name: lint
+
+steps:
+- name: lint
+  image: miki725/pre-commit
+
+trigger:
+  ref:
+    - refs/heads/master
+    - refs/pull/*/head
+
+---
+kind: pipeline
+name: deploy
+
+steps:
+- name: deploy
+  image: docker/compose:1.24.1
+  environment:
+    COMPOSE_PROJECT_NAME: sda-react
+    MOUNT_PATH: /tmp/app
+  volumes:
+    - name: docker_sock
+      path: /var/run/docker.sock
+  commands:
+    - docker-compose build --pull
+    - docker-compose up -d
+
+trigger:
+  ref:
+    - refs/heads/master
```

Comment:
on every master push it will deploy now on my server:

https://firstrussian.miki725.com